docs/docs/recipes/create-your-connector/platform-types.mdx

```diff
@@ -0,0 +1,15 @@
+# Platform types
+
+Social connectors' platforms can be divided into three types: Web, Native and Universal.
+
+## Web
+
+Social connectors with _Web_ platform type are build for desktop web applications.
+
+## Native
+
+_Native_ type social connectors are specially for those cases that users are required to go through authorization process in corresponding native mobile apps.
+
+## Universal
+
+_Universal_ type social connectors can work smoothly not only on desktop web applications, but also on mobile web applications via browser.
```

Comment:
Maybe they should be listed in `reference` but not `recipes`?

Reply:
Temporarily move connector platform type explanation to connector reference and add a TODO to build the link after finishing the connector reference part.